dockerfiles/compilation/java-8/compile.sh

```diff
@@ -56,7 +56,7 @@ git checkout $SHA1
 if [ "$SKIPTESTS" = "skipTests" ]; then
    mvn package -DskipTests ${MVN_ADDITIONAL_ARG_LINE}
 else
-   mvn package ${MVN_ADDITIONAL_ARG_LINE}
+   mvn package -DskipTests ${MVN_ADDITIONAL_ARG_LINE}
```

Comment:
this command it looks same to the above command, then `$SKIPTESTS` variable become useless?

Reply:
That was to forcefully skip the tests and test just the deployment